spec/services/schedule_service_spec.rb

```diff
@@ -18,14 +22,38 @@
     end
 
     it "creates a timeline entry" do
+      stub_default_publishing_api_put_intent
+
       edition = create(:edition)
       ScheduleService.new(edition).schedule
 
       expect(edition.timeline_entries.first.entry_type).to eq("scheduled")
     end
 
+    it "makes a request to publishing-api to create / update a publishing intent" do
+      publish_time = Time.current.tomorrow.at_noon
+      edition = create(:edition, scheduled_publishing_datetime: publish_time)
```

Comment:
Since this seems to be the only class where we're concerned with the `document_type.publishing_metadata.rendering_app` we might want to create our document type in this test rather than change the factory.

Then we could also avoid the big chain of `edition.document_type.publishing_metadata.rendering_app`

Reply:
Yep, see the above comment.